app/src/main/java/com/github/libretube/ui/fragments/HomeFragment.kt

```diff
@@ -75,13 +78,13 @@ class HomeFragment : Fragment() {
     }
 
     private fun fetchHomeFeed() {
-        lifecycleScope.launchWhenCreated {
```

Comment:
The goal here has been to load some of them simultaneously in order to benefit from faster loading times in the home page in general. Thus, I don't think this should be changed.

Reply:
In that case, I could run them concurrently using `async` blocks.